File: talker_listener_exercise/src/talker_listener_exercise/talker_node.py
#!/usr/bin/env python

import rospy

class Talker:
    def __init__(self):
        rospy.loginfo('talker_node: Hello World')

    def publish(self):
        pass

if __name__ == '__main__':
    rospy.init_node('talker_node', anonymous=False)
    talker = Talker()
    r = rospy.Rate(5)
    while not rospy.is_shutdown():
        talker.publish()
        r.sleep()

File: talker_listener_exercise/src/talker_node.cpp
#include "talker_listener_exercise/talker.h"

int main(int argc, char** argv)
{
  ros::init(argc, argv, "talker_node");
  talker_listener_exercise::Talker talker;
  ros::Rate r(5);
  while (ros::ok())
  {
    talker.publish();
    ros::spinOnce();
    r.sleep();
  }
}
